networking/multiple_networks/understanding-user-defined-network.adoc

```diff
@@ -1,71 +0,0 @@
-:_mod-docs-content-type: ASSEMBLY
-[id="understanding-user-defined-networks"]
-= Understanding user-defined networks
-include::_attributes/common-attributes.adoc[]
-:context: understanding-user-defined-networks
-
-toc::[]
-
-:featurename: `UserDefinedNetwork`
-include::snippets/technology-preview.adoc[]
-
-The OVN-Kubernetes CNI plugin supports `layer2`, `layer3`, and `localnet` topologies for secondary pod networks. However, for the primary network, or the main network that all pods are attached to, only a `layer3` topology is supported. This allows for network models where all pods in the cluster were part of the same global `layer3` network, but restricts the ability to customize primary network configurations.
```

Comment:
This was revised. It is on this page: https://84239--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/multiple_networks/primary_networks/about-user-defined-networks

I revised this comment because at least two people messaged me talking about how it was a confusing lead-in paragraph. Or got the impression that we were saying we supported Localnet, and Brandon Sousza also found it confusing (you, him, and I were in a thread together and talked about it). I hope that the revision is slightly clearer.